internal/static/metrics/agent.go

```diff
@@ -19,7 +19,7 @@ import (
 var DefaultConfig = Config{
 	Global:                 instance.DefaultGlobalConfig,
 	InstanceRestartBackoff: 5 * time.Second,
-	WALDir:                 "data-agent/",
+	WALDir:                 "data-alloy/",
```

Comment:
This one is intentional; internal/static is kept around for the config converters from static -> Alloy, and so we want to keep its default configuration the same, where it does write to data-agent/ by default :)

Reply:
Maybe adding a comment instead would be useful here so others can more easily understand why it’s not renamed